Marlin/src/lcd/marlinui.cpp

```diff
@@ -1692,9 +1692,10 @@ constexpr uint8_t epps = ENCODER_PULSES_PER_STEP;
     const PauseMode mode/*=PAUSE_MODE_SAME*/,
     const uint8_t extruder/*=active_extruder*/
   ) {
-    if (mode == PAUSE_MODE_SAME)
-      return;
+    //if (mode == PAUSE_MODE_SAME) return;
+    SERIAL_ECHOLNPAIR("PauseMsg=", message);
     pause_mode = mode;
+    ExtUI::pauseModeStatus = message;
```

Comment:
I see that `ExtUI::pauseModeStatus` is not used anywhere. What is the plan for its future use?

Reply:
That one is intended to be used by the screen implementation of it does not have its own messaging defined. Case on that for can messages, or following the status struct for custom handling.

Reply:
Would you like to re-add it and include its usage in one of the screen implementations?